tools/test-tools/package.json

```diff
@@ -1,6 +1,7 @@
 {
-	"name": "@fluidframework/test-tools",
-	"version": "3.0.0",
+	"name": "@fluid-private/test-tools",
+	"version": "2.22.0",
```

Comment:
Not for this PR, but is there a reason we use "real" version numbers for our `"private": true` packages? I think some tools assume this property _exists_, or I would suggest we remove them altogether. But it seems like we at least shouldn't need to be bumping them alongside published packages, and they could just be `0.0.0` or something.

Reply:
I think it's just that the policy checks want all packages in the release group to have matching versions. I agree it might be an improvement to instead check for `0.0.0` for `private: true` packages. cc @tylerbutler in case he knows of some edge case in our infra or something.